compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -413,66 +404,31 @@ def create_batch(self, task_group_id=None, create_websocket_queue=False) -> Batc
         Returns
         -------
         Batch instance
-            Status block containing "status" key.
         """
-        if not task_group_id:
-            task_group_id = self.session_task_group_id
-
-        return Batch(
-            task_group_id=task_group_id, create_websocket_queue=create_websocket_queue
-        )
+        return Batch(endpoint_id, task_group_id, create_websocket_queue)
 
     @requires_login
-    def batch_run(self, batch) -> t.List[str]:
-        """Initiate a batch of tasks to Globus Compute
+    def batch_run(self, batch: Batch) -> dict:
+        """
+        Initiate a batch of tasks to Globus Compute
 
-        Parameters
-        ----------
-        batch: a Batch object
+        :param batch: a Batch object
 
         Returns
         -------
-        task_ids : a list of UUID strings that identify the tasks
+        dictionary with the following keys:
+            tasks: a dictionary mapping function IDs to lists of tasks associated with
+                that function
```

Comment:
Redundant; suggest just:
```diff
- tasks: a dictionary mapping function IDs to lists of tasks associated with
-     that function
+ tasks: a mapping of function IDs to lists of task IDs
```

You might also consider reinforcing the API note that the the returned task IDs are in the order submitted to each function.  (We haven't published those REST API docs yet, but RSN.)